mvc/src/main/java/nextstep/mvc/view/JspView.java

```diff
@@ -1,30 +1,45 @@
 package nextstep.mvc.view;
 
+import jakarta.servlet.RequestDispatcher;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
+import java.util.Map;
+import java.util.Objects;
+import nextstep.mvc.exception.NotFoundException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.Map;
-
 public class JspView implements View {
 
     private static final Logger log = LoggerFactory.getLogger(JspView.class);
+    private static final String REDIRECT_PREFIX = "redirect:";
 
-    public static final String REDIRECT_PREFIX = "redirect:";
+    private final String viewName;
 
-    public JspView(final String viewName) {
+    private JspView(final String viewName) {
+        this.viewName = viewName;
     }
 
-    @Override
-    public void render(final Map<String, ?> model, final HttpServletRequest request, final HttpServletResponse response) throws Exception {
-        // todo
+    public static JspView from(final String location) {
+        if (location.startsWith(JspView.REDIRECT_PREFIX)) {
```

Comment:
```suggestion
        if (location.startsWith(REDIRECT_PREFIX)) {
```
현재 같은 클래스 안의 상수를 참조하고 있어요! 이렇게 바꿔보는건 어떨까요? 😄

Reply:
JspView 내부로 가져오면서 미처 지우지 못했나 보네여 😅